### ISSUE TITLE
'button links' don't work in Firefox

### ISSUE THREAD

**Issue by raboof** [grpc/grpc.github.io#746]
The 'button links', i.e. on https://grpc.io/contribute, don't seem to work in Firefox (I'm on version 61.0.1)

**Comment by carl-mastrangelo** [comment]
@raboof I see some of the buttons are missing text, but the ones with text seem to work.  Are you clicking directly on the text?

**Comment by raboof** [comment]
Yes - like VIEW GUIDELINES and BROWSE THIS SITE'S SOURCE CODE. I *guess* it could be related to the Vimium plugin I'm using - though I haven't seen such problems with it before. (interestingly I *can* click the link through Vimium, actually :) )
